src/rest/AnalyticsEndpoint.ts

```diff
@@ -93,22 +95,40 @@ export class AnalyticsEndpoint {
     return this.getFromService<string[]>(url, params);
   }
 
+  public forgetVisitorId() {
+    Cookie.erase('visitorId');
+  }
+
+  public kill() {
```

Comment:
You can use the usual disable() and disabled instead of dead/kill it's not terms we really use in that code base.

Reply:
This is resolved as I removed the ability to disable [AnalyticsEndpoint](../tree/master/src/rest/AnalyticsEndpoint.ts) directly.